ui/pages/confirm-add-suggested-token/index.scss

```diff
@@ -1,9 +1,77 @@
 .confirm-add-suggested-token {
+  padding: 16px;
+
   &__link {
     @include H7;
 
     display: inline;
     color: var(--primary-blue);
     padding-left: 0;
   }
+
+  &__header {
+    @include H7;
+
+    display: flex;
+  }
+
+  &__token {
+    flex: 1;
+    min-width: 0;
+  }
+
+  &__balance {
+    flex: 0 0 30%;
+    min-width: 0;
+  }
+
+  &__token-list {
+    display: flex;
+    flex-flow: column nowrap;
+
+    .token-balance {
+      display: flex;
+      flex-flow: row nowrap;
+      align-items: flex-start;
+
+      &__amount {
+        @include H1;
+
+        color: var(--scorpion);
+        margin-right: 8px;
+      }
+
+      &__symbol {
+        @include H5;
+
+        color: var(--scorpion);
+      }
+    }
+  }
+
+  &__token-list-item {
+    display: flex;
+    flex-flow: row nowrap;
+    align-items: center;
+    margin-top: 8px;
+    box-sizing: border-box;
+  }
+
+  &__data {
+    display: flex;
+    align-items: center;
+    padding: 8px;
+  }
+
+  &__name {
+    min-width: 0;
+    white-space: nowrap;
+    overflow: hidden;
+    text-overflow: ellipsis;
+  }
+
+  &__token-icon {
+    margin-right: 12px;
```

Comment:
`margin-inline-end` would be good for RTL.

Reply:
@darkwing  
`margin-inline-end` is new to me. I see the benefit when the direction changes. How does it help here?

Reply:
Update: spoke with @darkwing about this and we don't think margin-inline-end is necessary here